decidim-design/app/controllers/decidim/design/foundations_controller.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Design
+    class FoundationsController < Decidim::Design::ApplicationController
+      include Decidim::ControllerHelpers
+
+      def show
+        render "decidim/design/foundations/#{params[:id]}"
```

Comment:
This generates a 500 error when the template doesn't exist: 
http://localhost:3000/design/foundations/foo
It should be a 404.

Reply:
Done